tests/framework/utils/k8s_helper.go

```diff
@@ -720,7 +681,7 @@ func (k8sh *K8sHelper) GetRookLogs(podAppName string, namespace string, testName
 				continue
 			}
 		}
-		fileName := testName + "_" + podName + "_" + strconv.FormatInt(time.Now().Unix(), 10) + ".log"
+		fileName := testName + "_" + podName + "_" + namespace + "_" + strconv.FormatInt(time.Now().Unix(), 10) + ".log"
```

Comment:
another place where `fmt.Sprintf()` would be more readable. For the time you would only need `%d` instead of the `FormatInt()`